tests/testthat/test-generate-dust-sexp.R

```diff
@@ -89,6 +92,25 @@ test_that("can use functions from the library", {
 })
 
 
+test_that("can generate min/max", {
+  dat <- generate_dust_dat(c(a = "state",
+                             b = "stack",
+                             c = "shared",
+                             d = "internal",
+                             e = "data"))
```

Comment:
We don't have tests below that use a-e, but all covered above.

Reply:
yeah, this was just setup copied from above